values/opensuse.yaml

```diff
@@ -1,24 +0,0 @@
-image: opensuse/leap:15.2
```

Comment:
I'd keep this around (maybe by changing its name) as an example on how to apply templates. It is also required in case someone wants to rebuilds off entirely all the images

Reply:
Yes I have been thinking about it, and I was wondering if it wouldn't be easier to allow `values` to be split in multiple files. So for instance we could combine the base `values` provided by cOS and then user add/override values with an extra yaml file. 

What I don't like of keeping it around is that then we are kind a forking the original and it could eventually cause hard to notice inconsistencies on cOS base upgrades. So if we keep the original from the submodule we know it is in sync with the cOS base and additional tweaks could be applied at package level (most likely the top level one).

Reply:
makes sense, I'll open an issue to allow multiple values file to be applied during build :+1: maybe we can just retain a values file for the local tweakings as an example, without copying the content from the original cOS values files